content/article/quoting-the-shell.md

```diff
@@ -156,7 +156,9 @@ foreach my $file ( @ARGV ) {
 	}
 ```
 
-Blerg. That works but is ugly in the service of keystrokes (but how many actual keystrokes did I use to get to the final result?). I can open a pipe to the command and specify the command and its arguments as a list. This requires neither quoting nor escaping anything because each argument in Perl is one argument in the command (like [system](https://perldoc.perl.org/functions/system.html) in its list form):
+Blerg. That works in this case but is ugly in the service of keystrokes (but how many actual keystrokes did I use to get to the final result?). And, it probably misses some other special case.
```

Comment:
If you want a more concrete example of what can break this solution, some examples would be filenames containing `$` or ` (backtick).